app/src/store/voice/accessToken.ts

```diff
@@ -75,7 +89,7 @@ export const getAccessToken = createTypedAsyncThunk<
 
 export type AccessTokenState = AsyncStoreSlice<
   { value: string },
-  GetAccessTokenRejectValue | { error: any }
+  GetAccessTokenRejectValue | { error: any; reason: 'unknown' }
```

Comment:
```suggestion
  GetAccessTokenRejectValue | { error: any; reason: 'UNKNOWN_ERROR_OCCURRED' }
```

Reply:
Maybe not this exact wording, but prefer all caps and underscores between words to keep consistency.